File: app/src/main/java/org/mifos/mobilewallet/account/domain/model/Account.java
package org.mifos.mobilewallet.account.domain.model;

/**
 * Created by naman on 11/7/17.
 */

public class Account {

    private String image;
    private String name;
    private String number;
    private long id;

    public String getImage() {
        return image;
    }

    public void setImage(String image) {
        this.image = image;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getNumber() {
        return number;
    }

    public void setNumber(String number) {
        this.number = number;
    }

    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }
}

File: core/src/main/java/org/mifos/mobilewallet/core/domain/model/Account.kt
package org.mifos.mobilewallet.core.domain.model

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

/**
 * Created by naman on 11/7/17.
 */
@Parcelize
data class Account(
        var image: String? = null,
        var name: String? = null,
        var number: String? = null,
        var balance: Double? = null,
        var id: Long? = null,
        var productId: Long? = null,
        var currency: Currency? = null) : Parcelable